public/javascript/controllers/user-ctrl.js

```diff
@@ -86,6 +86,30 @@ angular.module('harvestv2')
 
         }
     ]
+).controller("DocsCtrl", ['$scope', '$location', '$routeParams', 'UserFactory', 'RolesFactory',
+        'PlatformFactory', 'CurrentUserFactory',
+        function($scope, $location, $routeParams, UserFactory, RolesFactory, PlatformFactory, CurrentUserFactory) {
+
```

Comment:
@thinkking writing jQuery within angular framework is bad.  Attempt to rewrite this controller using Angular functions.